src/main/java/lotto/model/LottoTicket.java

```diff
@@ -0,0 +1,49 @@
+package lotto.model;
+
+import java.util.Comparator;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class LottoTicket {
+
+    public static final Long PRICE = 1_000L;
+    public static final int SIZE = 6;
+    private final List<LottoNumber> lottoNumbers;
+
+    public LottoTicket(List<LottoNumber> lottoNumbers) {
+        validate(lottoNumbers);
+        this.lottoNumbers = lottoNumbers.stream()
+                .sorted(Comparator.comparing(LottoNumber::getNumber))
+                .collect(Collectors.toList());
+    }
+
+    private void validate(List<LottoNumber> lottoNumbers) {
+        validateSize(lottoNumbers);
+        validateDuplication(lottoNumbers);
+    }
+
+    private void validateDuplication(List<LottoNumber> lottoNumbers) {
+        if (lottoNumbers.stream().distinct().count() != SIZE) {
```

Comment:
적당히 변수로 뽑아주는 것이 조금 더 가독성이 좋아질 것 같아요. 🙂
```suggestion
        int distinctLottoNumbersSize = lottoNumbers.stream()
            .distinct()
            .count()
        if (distinctLottoNumbersSize != SIZE) {
```

혹은 중복을 허용하지 않는 `Set` 자료구조를 이용하는 방법도 생각해볼 수 있을 것 같네요. 😉

Reply:
SortedSet과 Compartor이용하게 변경하였습니다!